### DISCUSSION TITLE
Automatically create regions (organization)

### DISCUSSION THREAD

**Discussion by 22Jean-Paul** [netbox-community/netbox#12578] [Help Wanted!] [answered]
Hello NetBox community,
I need your help to find out how
automatically create 3 continents, 2 countries per continent and 2 cities per country following the example below

For ASIA
1) China (Beijing, Shanghai)
2) Japan (Tokyo, Osaka)

For AMERICA
1) United States (New York, Los Angeles)
2) Canada (Toronto, Montreal)

For EUROPE)
1) France (Paris, Lyons)
2) United Kingdom (London, Manchester)

Thank you in advance for your assistance

**Comment by candlerb** [comment, marked as answer]
You can import regions via CSV. Go to Organizations > Regions > (Upload button) and it will describe the format for you, and you can paste the CSV directly into the box.

```
name,slug,parent
Asia,asia,
Japan,japan,Asia
Tokyo,tokyo,Japan
Osaka,osaka,Japan
... etc
```

**Reply by 22Jean-Paul** [reply]
Thanks à lot.
